### DISCUSSION TITLE
[Exercise-11] Create tables FDS_ITEMS and FDS_RESTAURANTS with PRIMARY and FOREIGN keys

### DISCUSSION THREAD

**Discussion by acrlakshman** [exaslate-courses/oracle-sql-plsql-b3#13] [General]
* Create `FDS_ITEMS` first, and observe the error due to missing `FDS_RESTAURANTS`. Please share a screenshot of this error.
  * This was mentioned correctly in the lecture at first, but then I misrepresented when it accidentally worked.

* Create FDS_ITEMS with the following members
  * ID NUMBER and PRIMARY KEY of this table. Use ORA_HASH to populate default value.
  * RESTAURANT_ID NUMBER and FOREIGN KEY of member FDS_RESTAURANTS.ID
  * NAME VARCHAR2(30)
  * PRICE NUMBER
  * CREATED_DATE
  * MODIFIED_DATE

* Create FDS_RESTAURANTS with the following members
  * ID NUMBER and PRIMARY KEY of this table. Use ORA_HASH to populate default value.
  * NAME VARCHAR2(30)
  * CREATED_DATE
  * MODIFIED_DATE

Ref.: https://www.youtube.com/watch?v=5AUuWgMi8Sk&t=2940s

**Comment by srideviadapa0511** [comment]
![prmry](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/3bdaadd6-dc57-4826-b14f-41f2d3097151)

**Comment by srideviadapa0511** [comment]
![idly,dosa](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/f6968869-39a3-4512-86f3-8495d170143e)

**Comment by srideviadapa0511** [comment]
![idly,dossa,puri](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/f8ed5a8f-91d8-4a04-be84-0c6f47e0b524)

**Reply by acrlakshman** [reply]
@srideviadapa0511 nice work.

**Comment by Tharun1596** [comment]
![image](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/141386177/8e146dc2-ab91-4468-8a9c-6688919bb1fd)

**Comment by Tharun1596** [comment]
![image](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/141386177/75e8030f-8075-486f-b65f-769de7b8453f)
